docs/running-kics.md

```diff
@@ -86,3 +88,46 @@ Please make sure you have set GSC authentication credentials to your application
 
 More informatition can be seen [here](https://github.com/hashicorp/go-getter#gcs-gcs)
 
+## Using custom input data
+Since from v1.3.5, KICS supports using custom input data to replace data on queries that has this feature supported. To check if a query supports overwriting, just check if query's folder contains `data.json` file, this file will contain all keys that can be overwritten.
```

Comment:
```suggestion
Since from v1.3.5, KICS supports using custom input data to replace data on queries that have this feature supported. To see if a query supports overwriting, check if the query's folder contains a `data.json` file, this file will contain all keys that can be overwritten.
```